source/usage-examples/command.txt

```diff
@@ -0,0 +1,48 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. Call the ``runCommand()`` method
+with your command object on an instance of a database for diagnostic and
+administrative tasks such as fetching server stats or initializing a
+replica set.
+
+.. note::
+    Use the :manual:`mongo shell </reference/mongo-shell>` for
+    administrative tasks instead of the Java Driver whenever possible.
+
+The ``runCommand()`` method accepts a :manual:`command
+<reference/command/>` in Bson format. By default, ``runCommand`` returns
```

Comment:
Let's not link to the manual until the end as this is "external" to the Java driver docs. Instead mention "for more information on commands, see the link at the bottom of this page."